_posts/2020-07-14-newpipe-0.19.6-release.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "NewPipe 0.19.6 released: Audio/video desync bug and search improvements"
+short: "NewPipe 0.19.6 released"
+date: 2020-07-14 00:00:00 +0100
+categories: [release, pinned]
+author: gkeegan
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+No more unsynced videos! This release will let your videos make sense and improve searching in NewPipe to make it more reliable and widespread.
+
+<!-- more -->
+
+## Additions
+
+You can now add a playlist directly to your main page as a tab, a great change for those who frequently use NewPipe for music. <sup>[@Royosef in #3506](https://github.com/TeamNewPipe/NewPipe/pull/3506)</sup>
+
+There is an option to prevent subscriptions from being visible in the subscription picker for feeds which will be helpful to sort your subscriptions into different categories. <sup>[@mauriciocolli in #3404](https://github.com/TeamNewPipe/NewPipe/pull/3404)</sup>
```

Comment:
Maybe clarify that subscriptions that are already in another feed won't be shown with this option enabled.

Reply:
That doesn't occur when they are in the default feed, correct?

Reply:
Of course the default "All" feed is excluded, as that always contains all channels and can't be changed

Reply:
Ok good. Then this should do the trick:
```suggestion
There is an option to prevent subscriptions from being visible in the subscription picker for feeds which will be helpful to sort your subscriptions into different categories. If enabled, subscriptions that are already in another non-default feed will not be shown. <sup>[@mauriciocolli in #3404](https://github.com/TeamNewPipe/NewPipe/pull/3404)</sup>
```

Reply:
That still isn't that clear imho, I was thinking more like the following: "There now is an option in the subscription picker to not show channels that are already in any feed (other than "All")."